code/modules/organs/organ_icon.dm

```diff
@@ -79,8 +81,14 @@ var/global/list/limb_icon_cache = list()
 		//We have weird other-sorts of eyes (as we're not supposed to have eye organ, but we have HAS_EYE_COLOR species)
 		else
 			eyes_icon.Blend(rgb(owner.r_eyes, owner.g_eyes, owner.b_eyes), ICON_ADD)
-		add_overlay(eyes_icon)
-		mob_icon.Blend(eyes_icon, ICON_OVERLAY)
+
+		//VOREStation edit -- allow rendering of eyes over markings.
```

Comment:
Would this mess with the heterochromia marking?

Reply:
Yes, but if you want to render your eyes above markings that is a caveat that must be understood-- eyes/monitor styles will render above all markings

However if your 'eye'/monitor sprite is largely transparent, or if the eye sprite only has pixels for the one eye to begin with, then the heterochromia will still render-- it won't be clipped out by the transparency of the eye sprite above